From 11ee1366ac88cf45281440a16b1b1f05da6e138d Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Mon, 12 Nov 2018 07:56:15 +0000
Subject: [PATCH] Revert "imx: mx7: spl: Support mask 3N09P of i.MX7 ("revision
 D")"

This reverts commit 466898df5360675516bf1f014e5d48e507ae5c95.

Reverted as it breaks boot with imx7 revisions older than D 

Upstream-Status: Innappropriate[revert]

Author: Eran Matityahu <eran.m@variscite.com>
Signed-off-by: Eran Matityahu <eran.m@variscite.com>
---
 include/configs/imx7_spl.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/imx7_spl.h b/include/configs/imx7_spl.h
index 8c8c327..b89dba6 100644
--- a/include/configs/imx7_spl.h
+++ b/include/configs/imx7_spl.h
@@ -25,7 +25,7 @@
  *    and some padding thus 'our' max size is really 0x00946BB8 - 0x00911000.
  *    64KB is more then enough for the SPL.
  */
-#define CONFIG_SPL_TEXT_BASE		0x00912000
+#define CONFIG_SPL_TEXT_BASE		0x00911000
 #define CONFIG_SPL_MAX_SIZE		0x10000
 #define CONFIG_SPL_STACK		0x00946BB8
 /*
-- 
2.7.4
+
